test/Altinn.Platform.Events.Tests/TestingControllers/EventsControllerTests.cs

```diff
@@ -64,8 +62,11 @@ public EventsControllerTests(WebApplicationFactory<EventsController> factory)
                     Id = Guid.NewGuid().ToString(),
                     Type = "system.event.occurred",
                     Subject = "/person/16069412345",
-                    Source = new Uri("urn:isbn:1234567890"),
+                    Source = new Uri("urn:isbn:1234567890")
                 };
+
+                _validEvent["resource"] = "urn:altinn:rr:nbib.bokoversikt.api";
```

Comment:
Did we agree on "urn:altinn:rr" or "urn:altinn:resoure"? I thought I was sure, but now I'm not.

Reply:
Not quite sure about that. Given that its only testdata and the value does affect the functionalty I didn't research. resource is correct we can update.